### DISCUSSION TITLE
API (from Gamestonk Terminal)

### DISCUSSION THREAD

**Discussion by KhanPhelan** [agrlh/fininsights#9] [Ideas]
https://github.com/GamestonkTerminal/GamestonkTerminal#api-keys

**Comment by ZuitAMB** [comment]
Convenient API for stock data, fundamentals etc. (300req/day free, more costs more)
https://financialmodelingprep.com/developer/docs

Yahoo Finance (Webscaping or API)
https://rapidapi.com/blog/how-to-use-the-yahoo-finance-api/

**Comment by ZuitAMB** [comment]
More Yahoo APIs
https://github.com/ranaroussi/yfinance
https://github.com/dpguthrie/yahooquery
